src/functionaltests/resources/features/rabbitMQConfiguration.feature

```diff
@@ -5,4 +5,4 @@ Feature: Test Rabbit MQ Configuration
   Scenario: Test that aggregations are done when events are sent with different routing keys
     Given We are connected to message bus
     When events are published using different routing keys
-    Then an aggregated object should be created
\ No newline at end of file
+    Then an aggregated object should be created
```

Comment:
I guess, to really be transparent I think we should mention which aggregation we expect to see (i.e. which content it should have). If it looks as expected we know that all events have been aggregated enough. It is not clear right now if that comparison is done (since it's hidden in a separate method).

Reply:
So you mean to compare the complete object then? Atm I just reused an old method to find specific fields.

Reply:
Perhaps it is enough to compare specific fields, the reason I asked was because it wasn't clear when I just skimmed through the changes in the tests ... But as long as the aggregation content is verified then I'm satisfied.

Reply:
I check that the aggregation has the id of the start event and that part of the publish information was aggregated as well.

Reply:
Alright, then I'm happy. Approving this.